src/components/header/Header.jsx

```diff
@@ -0,0 +1,15 @@
+import style from './header.module.scss';
+import InfoPanel from '../InfoPanel/InfoPanel';
+import Search from '../search_form/Search';
+
+function Header({ onSearch, isLoading, ip, location, timezone, isp }) {
+  return (
+    <div className={style.finder}>
+      <h1 className={style.title}> IP Address Tracker</h1>
```

Comment:
Why did you add a space inside h1 before text?

Reply:
oops